darwin/cli_functions.py

```diff
@@ -830,22 +832,23 @@ def dataset_convert(dataset_slug: str, format: str, output_dir: Optional[PathLik
         The folder where the exported annotation files will be. If None it will be the inside the
         annotations folder of the dataset under 'other_formats/{format}'. The Defaults to None.
     """
-    client: Client = _load_client()
+    identifier: DatasetIdentifier = DatasetIdentifier.parse(dataset_slug)
+    client: Client = _load_client(team_slug=identifier.team_slug)
     exporter_module = import_module(f"darwin.exporter.formats.{format}")
     try:
         parser: ExportParser = getattr(exporter_module, "export")
     except AttributeError:
         _error(f"Unsupported import format, currently supported: {export_formats}")
 
     try:
-        dataset: RemoteDataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
+        dataset: RemoteDataset = client.get_remote_dataset(dataset_identifier=identifier.dataset_slug)
         if not dataset.local_path.exists():
             _error(
                 f"No annotations downloaded for dataset f{dataset}, first pull a release using "
-                f"'darwin dataset pull {dataset_slug}'"
+                f"'darwin dataset pull {identifier}'"
```

Comment:
Will mypy complain here?
```suggestion
                f"'darwin dataset pull {str(identifier)}'"
```

Reply:
you shouldn't need that, and identifier implements `__str__`